### ISSUE TITLE
Update in A2S_INFO packet (added challenge)

### ISSUE THREAD

**Issue by overfl0** [serverstf/python-valve#85]
Hi!

I realize that you said that this library will not be maintained anymore, but just to let you know, valve will be implementing an (optional for now) challenge for A2S_INFO.

The feature is currently being tested and details can be found here: https://steamcommunity.com/discussions/forum/14/2974028351344359625/

Since this is a change that will completely break the current library the moment the challenge becomes mandatory, it would be nice if you could implement this as a last change and push it to pypi as well so that the library could still be used in the future without the need to rewrite old software that relies on it.

The TL;DR is that the first A2S_INFO request stays the same and if the response contains a challenge, you simply append it to the request and resend. In other words, it works just like with A2S_PLAYER but instead of passing 0xffffffff, you just send nothing, the first time.

**Comment by overfl0** [comment]
Also, at the end of the first page of the steam discussion, there is someone who posted their server IP and port for testing

**Comment by Yepoleb** [comment]
The challenge is already implemented in python-a2s. I have no interest in doing double the work for a few people who can't be bothered to update a few lines of their code. Also nobody apart from the owner has access to the official PyPi package, so it would have to be published under a new name, requiring old software to be manually patched anyway.
